src/components/AttractionCard.js

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components/macro'
+
+export default function AttractionCard({ name, image }) {
+  return (
+    <Card key={name}>
+      <AttractionName>{name}</AttractionName>
+      <Image src={image} width="300" height="auto" alt=""></Image>
```

Comment:
It's recommended to set width and height in total values without a unit, to give the browser a hint on which aspect ratio the image will have. Please don't set it to "auto".